assignment4/index.html

```diff
@@ -74,25 +74,36 @@
                 <div class="date"><p>3</p></div>
             </section>
             <!-- <section class="calendar"></section> -->
-            <section class="main">
-                <ul>
-                    <h2>재훈</h2>
-                    <li></li>
+            <main>
+                <ul class="main_ul">
+                    <section class=""><h2>인간관계</h2></section>
+                    <li>친구들 자주 만나기</li>
+                    <li>좋은 말 많이 해주기</li>
                 </ul>
-                <ul>
-                    <h2>SOPT</h2>
-                    <li></li>
+                <ul class="main_ul">
+                    <section><h2>SOPT</h2></section>
+                    <li>잘 융화되기</li>
+                    <li>웹 심화 스터디 버티기</li>
+                    <li>잊지못할 추억 만들기</li>
                 </ul>
```

Comment:
```suggestion
<article>
               <header><h2>SOPT</h2></header>
               <ul class="main_ul">
                     <li>잘 융화되기</li>
                    <li>웹 심화 스터디 버티기</li>
                    <li>잊지못할 추억 만들기</li>
                </ul>	
 </article>                
```
로 작성하면 조금더 알맞는 시멘틱 태그 사용법 일것같아...!